src/main/java/uk/gov/hmcts/reform/ccd/document/am/service/impl/CaseDataStoreServiceImpl.java

```diff
@@ -98,13 +98,12 @@ public CaseDataStoreServiceImpl(RestTemplate restTemplate, SecurityUtils securit
         return Optional.empty();
     }
 
-    private HttpHeaders prepareRequestForUpload(String authorization) {
+    private HttpHeaders prepareRequestForUpload() {
 
         HttpHeaders headers = new HttpHeaders();
         headers.addAll(securityUtils.authorizationHeaders());
         headers.setContentType(MediaType.APPLICATION_JSON);
         headers.set("experimental", "true");
-        headers.set("Authorization", authorization);
```

Comment:
Sending authorization header to ccd-data-store is mandatory, could you please get the header from spring security context and add here,

Reply:
I know it  mandatory and added the authorization header in SecurtityUtils.java.If you will  check securityUtils.authorizationHeaders(). it will give you same.